### DISCUSSION TITLE
TBC World Bosses, Badge of Justice Gear, and Future Content

### DISCUSSION THREAD

**Discussion by Cooties-cc** [chromiecraft/chromiecraft#5058] [General]
Think it was a better choice to open a discussion about these, rather than hope for a response in the general discord chat sea of posts. The topic is pretty straightforward, and has sorta been talked about or asked before, but without a direct visible answer to clear confusion. 

**Update on choice** : I have got an answer that classic TBC is the preferred timeline for changes. I will leave this up just to be safe to assume it is a final choice. 

To preface, it would be extremely helpful if the devs/GMs gave a direct and clear answer about how the progression is being decided going forwards into TBC content. How I see it, there are 2 roads which were : follow original TBC content directly relating to the progression and releases that happened *only* during TBC, or follow the changes made to the original version of the game that took place during classic TBC. Letting players know which direction is being chosen can clear up a ton of confusion over what is to be expected down the line of progression. Of course it doesn't have to be black and white, and some changes can be influenced by the staff or players. 

The first topic is addressing the release of TBC world bosses. Now in both original and classic, these were released during p1/2.0.3. This would be about the same for CC release timeline to drop them with the start of T4 boss, mag/gruul, or make custom change, and drop them with release of heroics. However there's been almost zero talk or demonstration that these are going to be available by then. The loot that drops from these 2 world bosses is extremely good for classes, and affects how people choose to gear up their character. I would appreciate if there was some sort of addressing to these two bosses by the end of next patch or at least a good time before T4 drops so people can plan accordingly. 

The second topic is the gear availability of the badge vendor. I have made a bug report showing that the vendor was following the classic TBC approach and including patch 2.1 gear, which was going against the grain of the progression branch that was added. It was confirmed true, however there is a large push against following original TBC timeline, and instead picking the classic TBC timeline. There has also been other events that display a favor towards classic TBC timeline from staff. It would be very helpful to clarify which direction the staff is going to choose, since again, this deeply affects how players are going to gear their characters, which could lead to crafted gear becoming obsolete. **(it is assumed classic is the choice)**

Future content addressing isn't a major thing, as it can be cleared up by which timeline we're going by. Since classic TBC prog is to be assumed the preferred choice, all content relating to the classic phases I would assume are going to be the acceptable changes in the future. This means things should be changed on the currently posted timeline like arena season 2 starting with SSC/TK release, however currently, it's slated for a hyjal/BT release which was phase 3 / season 3 going by classic TBC timeline. These are the small changes that needed to be addressed and clarified going into the future. As I said before too, it doesn't have to be black and white final verdicts, but it's extremely helpful to players to know which correct info should we be expecting and following.

**Comment by Cooties-cc** [comment]
![time](https://user-images.githubusercontent.com/62584589/219536533-8c927fce-308f-44d5-a060-9041ffc49182.png)

Forgot to add that this was the classic TBC timeline broad overview of content. For clarity, the world bosses were open 2 weeks after t4 dropped.

**Comment by B4nanaboi** [comment]
Good points by Cooties. I'd like to emphasise the importance of the worldboss unlock timing, because both Doomlord Kazzak and Doomwalker drop many, very strong ilvl 120 weapons and other major upgrades for nearly all classes. It's about Karazhan ilvl gear, but often BiS itemised (stat distribution/weaponspeed etc.). 

If we use classic TBC phasing as guideline, we still need to make a choice. Phase1 classic TBC has Gruul+Mag+Kara together, but on Chromie these are split in two different phases. It's good to know at which Chromie phase the world bosses unlock.

Badge gear: same thing. I'd love to know when I can get my Bloodlust Brooch :)

**Comment by balleny** [comment]
kinda overlapping with the unfortunately not answered ticket here:
https://github.com/azerothcore/mod-progression-system/issues/291

but, good to know that there apparently is a shift towards classic instead of vanilla progression - mind sharing some details who / when confirmed the shift towards classic?

**Reply by Cooties-cc** [reply]
![image](https://user-images.githubusercontent.com/62584589/220165155-fccfd635-b932-4789-b798-de4f070154e5.png)
So far only one to actually give somewhat of an answer to the prog issues.
